nix/.stack.nix/cardano-node.nix

```diff
@@ -4,8 +4,8 @@
     package = {
       specVersion = "1.10";
       identifier = { name = "cardano-node"; version = "3.0.1.87"; };
-      license = "MIT";
-      copyright = "2019 IOHK";
+      license = "NONE";
```

Comment:
Same here.

Reply:
That's wired, let me check.

Reply:
ok, I missed the `license` field in `cardano-node.cabal` file.